src/client/Microsoft.Identity.Client/ApiConfig/AbstractAcquireTokenParameterBuilder.cs

```diff
@@ -423,6 +489,25 @@ internal void ValidateAndCalculateApiId()
             CommonParameters.ApiId = CalculateApiEventId();
             CommonParameters.ApiTelemId = ApiTelemetryId;
             CommonParameters.CorrelationId = CommonParameters.UseCorrelationIdFromUser ? CommonParameters.UserProvidedCorrelationId : Guid.NewGuid();
+
+#if DESKTOP || NET_CORE
+            if (CommonParameters.UsingProofOfPossesion)
+            {
+                if (CommonParameters.PopUri == null )
+                {
+                    throw new MsalClientException(MsalError.PopUriCannotBeNull, "Proof of possesion endpoint is null.");
+                }
+
+                HttpRequestMessage message = new HttpRequestMessage(CommonParameters.PopMethod != null ? CommonParameters.PopMethod : HttpMethod.Get,
```

Comment:
No, this isn't how it's supposed to work. The user has the ability to say "don't bind the method to the pop token".